src/components/Masthead.js

```diff
@@ -18,6 +18,10 @@ type MastheadProps = {
   logoAlt: string | null;
   /** The route to navigate to when the user clicks the logo. Defaults to '/' */
   homeRoute: string | null;
+  /** The optional url to navigate to when the user clicks the logo. If homeRoute
+   *  and homeUrl are both specified, homeUrl takes precedence.
```

Comment:
Typically, when one of these comments spans multiple lines, we start it on the second line, leaving the first and last lines blank.
So the comment would look like
```
  /**
   * The optional url to navigate to when the user clicks the logo. If homeRoute
   * and homeUrl are both specified, homeUrl takes precedence.
   */
```
instead of
```
  /** The optional url to navigate to when the user clicks the logo. If homeRoute
   *  and homeUrl are both specified, homeUrl takes precedence.
   */
```

Reply:
Good to know, I'll update that. Once we update to the latest eslint version we can add this rule to our eslint config to enforce this:
https://eslint.org/docs/rules/multiline-comment-style#enforce-a-particular-style-for-multiline-comments-multiline-comment-style

Reply:
This convention is really only for JSDoc-style comments (and comes from similar conventions for JavaDoc comments). Apparently the ESLint folks are deprecating the current rules they have for JSDoc (see https://eslint.org/blog/2018/11/jsdoc-end-of-life) in favor of an ESLint plug-in to handle it (https://github.com/gajus/eslint-plugin-jsdoc). However, I don't see an option there for this sort of rule (e.g., enforcing that multi-line JSDoc comments have a newline right after the first /**... although there may be others we might want (e.g. newline-after-description or check-tag-names).

In any case, I don't think we want to use the multiline-comment-style rule because it doesn't differentiate between JSDoc comments and just plain block-style comments. It would, for example, complain about two consecutive lines with // comments... which, I think should be fine.

E.g.:
```
// this is a long comment
// separated into two lines
```

or
```
// This is one comment
// And this is another one that just happens to be adjacent to the first one
```

Anyway... :)

Reply:
Thanks for the clarification!